src/en/flamecomics/build.gradle

```
@@ -1,9 +1,7 @@
 ext {
     extName = 'Flame Comics'
     extClass = '.FlameComics'
-    themePkg = 'mangathemesia'
-    baseUrl = 'https://flamecomics.xyz'
-    overrideVersionCode = 4
+    extVersionCode = 5
```

```suggestion
    extVersionCode = 35
```

This will prompt existing users to update the extension. The value is +1 from the previous version:
```
baseVersionCode + overrideVersionCode =
30 + 4 = 34
// ext-lib is 1.4, so extension version is 1.4.34
```

You could also find the current version from https://keiyoushi.github.io/extensions/